crates/nu-cli/src/commands.rs

```diff
@@ -30,6 +30,7 @@ pub(crate) mod date;
 pub(crate) mod debug;
 pub(crate) mod default;
 pub(crate) mod do_;
+mod documentation;
```

Comment:
since there isn't a `documentation` command, you might want this to live somewhere more central. Maybe you could create a `nu-cli/src/documentation` directory.